driver/sql/postgres/eventstore_test.go

```diff
@@ -252,26 +254,14 @@ func TestEventStore_Load(t *testing.T) {
 			matcher       func() metadata.Matcher
 			expectedQuery string
 		}{
-			{
-				"With matcher",
-				1,
-				nil,
-				func() metadata.Matcher {
-					m := metadata.NewMatcher()
-					m = metadata.WithConstraint(m, "version", metadata.GreaterThan, 1)
-					m = metadata.WithConstraint(m, "version", metadata.LowerThan, 100)
-					return m
-				},
-				`SELECT \* FROM event_stream WHERE no >= \$1 AND metadata ->> 'version' > \$2 AND metadata ->> 'version' < \$3 ORDER BY no`,
-			},
```

Comment:
Question @boekkooi-fresh This test case should be updated to match `SELECT \* FROM event_stream WHERE no >= \$1 AND 'aggregate_version' > \$2 AND aggregate_version' < \$3 ORDER BY no`, right?

Reply:
@boekkooi-fresh :arrow_up:

Reply:
Yes this test case should be updated and not removed.

Reply:
https://github.com/hellofresh/goengine/pull/50/commits/c6d8685d15e6626ecc4cbd2239fd8be9fd3913f7